PyPrograms/Raspberry Pi Detector/requirements.txt

```diff
@@ -0,0 +1 @@
+os
```

Comment:
Here's a joke for whoever is looking in here. 
Q: Whats a computers favorite game?

A: OSu!